base/test/test_utils.cpp

```diff
@@ -110,12 +112,34 @@ bool Test_Utils::saveOrCompare(const char* fileName, const unsigned char* dataTo
 
 	if (boost::filesystem::is_regular_file(fileName))
 	{
+		const std::string strFullFileName = fileName;
+		std::string strFileBaseName = strFullFileName;
+		std::string strExtension;
+		std::string nameOfFile;
+
+		const size_t fileBaseIndex = strFullFileName.find_last_of(".");
+		const size_t fileBaseLocation = strFullFileName.find_last_of("/");
```

Comment:
test with simple filename with no / eg. file.raw

Reply:
Tested with only TestSample.raw it worked